packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -314,4 +314,36 @@ describe('masternode', () => {
       })
     })
   })
+
+  describe('getBlockStats', () => {
+    it('should get blockchain stats and return  all values', async () => {
+      const blockHash = await waitForBlockHash(1)
+      const stats = await client.blockchain.getBlockStats(blockHash)
```

Comment:
Stats return a long list of data. You should add assertions to all variables.

      avgfee: 0,
      avgfeerate: 0,
      avgtxsize: 0,
      blockhash: '19ae07a7bfae2cd8aefd34d2b6d764cf8d9cf0754230f6f2ebb512ce909d37f6',
      feerate_percentiles: [ 0, 0, 0, 0, 0 ],
      height: 1,
      ins: 0,
      maxfee: 0,
      maxfeerate: 0,
      maxtxsize: 0,
      medianfee: 0,
      mediantime: 1621916080,
      mediantxsize: 0,
      minfee: 0,
      minfeerate: 0,
      mintxsize: 0,
      outs: 3,
      subsidy: 5000000000,
      swtotal_size: 0,
      swtotal_weight: 0,
      swtxs: 0,
      time: 1621916080,
      total_out: 0,
      total_size: 0,
      total_weight: 0,
      totalfee: 0,
      txs: 1,
      utxo_increase: 3,
      utxo_size_inc: 239

Reply:
You miss these variables 

1. swtotal_weight
2. txs
3. minfeerate
4. mediantime

Reply:
Done @jingyi2811